File: Exercise/Easy/15. Digit Splitter/Code/Digit Splitter.java
import java.util.Scanner;

public class DigitSplitter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();
        digitSplitter(number);
    }

    public static void digitSplitter(int number) {
        String numStr = Integer.toString(number);
        for (char c : numStr.toCharArray()) {
            System.out.println(c);
        }
    }
}

File: Exercise/Easy/15. Digit Splitter/Code/Digit Splitter.cpp
#include <iostream>
#include <string>

using namespace std;

void digitSplitter(int number) {
    string numStr = to_string(number);
    for (char c : numStr) {
        cout << c << endl;
    }
}

int main() {
    int number;
    cin >> number;
    digitSplitter(number);
    return 0;
}

